redbot/cogs/general/general.py

```diff
@@ -218,35 +218,93 @@ def __init__(self):
+            "eu-west": _("EU West :flag_eu:"),
+            "eu-central": _("EU Central :flag_eu:"),
+            "london": _("London :flag_gb:"),
+            "frankfurt": _("Frankfurt :flag_de:"),
+            "amsterdam": _("Amsterdam :flag_nl:"),
+            "us-west": _("US West :flag_us:"),
+            "us-east": _("US East :flag_us:"),
+            "us-south": _("US South :flag_us:"),
+            "us-central": _("US Central :flag_us:"),
+            "singapore": _("Singapore :flag_sg:"),
+            "sydney": _("Sydney :flag_au:"),
+            "brazil": _("Brazil :flag_br:"),
+            "hongkong": _("Hong Kong :flag_hk:"),
+            "russia": _("Russia :flag_ru:"),
+            "japan": _("Japan :flag_jp:"),
+            "southafrica": _("South Africa :flag_za:")
+        }
         guild = ctx.guild
-        online = len([m.status for m in guild.members if m.status != discord.Status.offline])
+        online = len([m.status for m in guild.members if m.status == discord.Status.online])
+        idle = len([m.status for m in guild.members if m.status == discord.Status.idle])
+        dnd = len([m.status for m in guild.members if m.status == discord.Status.dnd])
+        offline = len([m.status for m in guild.members if m.status == discord.Status.offline])
         total_users = len(guild.members)
+        bots = len([a for a in ctx.guild.members if a.bot])
+        humans = len([a for a in ctx.guild.members if a.bot == False])
         text_channels = len(guild.text_channels)
         voice_channels = len(guild.voice_channels)
         passed = (ctx.message.created_at - guild.created_at).days
-        created_at = _("Since {date}. That's over {num} days ago!").format(
+        created_at = _("Created on **{date}**. That's over **{num}** days ago !").format(
             date=guild.created_at.strftime("%d %b %Y %H:%M"), num=passed
         )
+        joined_at = guild.me.joined_at
+        since_joined = (ctx.message.created_at - joined_at).days
+        bot_joined = joined_at.strftime("%d %b %Y %H:%M:%S")
+        joined_on = (_(f"{ctx.bot.user.name} joined this server on {bot_joined}. That's over {since_joined} days ago !"))
         data = discord.Embed(description=created_at, colour=(await ctx.embed_colour()))
-        data.add_field(name=_("Region"), value=str(guild.region))
-        data.add_field(name=_("Users"), value=f"{online}/{total_users}")
-        data.add_field(name=_("Text Channels"), value=str(text_channels))
-        data.add_field(name=_("Voice Channels"), value=str(voice_channels))
-        data.add_field(name=_("Roles"), value=str(len(guild.roles)))
-        data.add_field(name=_("Owner"), value=str(guild.owner))
-        data.set_footer(text=_("Server ID: ") + str(guild.id))
-
+        if total_users > 8000:
+            data.add_field(name=_("Members :"),
+            value=_(f"Total users : **{total_users}**\nHumans : **{humans}** • Bots : **{bots}**\n📗 `{online}` 📙 `{idle}`\n📕 `{dnd}` 📓 `{offline}`"))
```

Comment:
cant mix f-string with i18n translation like this, use `.format` (again, and at each additional instance)